be/src/olap/rowset/segment_v2/column_writer.cpp

```diff
@@ -382,12 +374,50 @@ Status ColumnWriter::_finish_current_page() {
     Page* page = new Page();
     page->first_rowid = _last_first_rowid;
     page->num_rows = _next_rowid - _last_first_rowid;
-    page->data = _page_builder->finish();
-    _page_builder->reset();
+    faststring header;
```

Comment:
why not encapsulate this in page->finish()

Reply:
I think it is ok to create head like this